File: code/online_challenges/src/project_euler/problem_001/problem_001.c
#include <stdio.h>

int
main(void)
{
    int sum = 0, i = 0;
    
    for (; i < 1000; ++i)
        sum += (i % 3 == 0) || (i % 5 == 0) ? i : 0;
    
    printf("%d\n", sum);
}

File: code/online_challenges/src/project_euler/problem_001/Problem_001.java
package Problem_001;
public class Problem_001 {

	public static void main(String []args) {
		int sum = 0;
		for(int i = 1; i < 1000; i++) {
			if(i % 3 ==0 || i % 5 == 0)
			{
				sum += i;
			}
		}
		System.out.print(sum); //233168
	} 
}
